src/NuGet.Services.Logging/project.json

```diff
@@ -1,8 +1,8 @@
 {
     "version": "2.0.0-rc2-3",

```

Comment:
yup, better to keep it consistent. Though when u call "dotnet pack" it uses the version here

---
In reply to: [74501062](https://github.com/NuGet/NuGet.Staging/pull/71#discussion_r74501062) [](ancestors = 74501062,74493268)